update-dois/pom.xml

```diff
@@ -0,0 +1,86 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.reactome.release.updateDOIs</groupId>
+  <artifactId>update-dois</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
Leave the version number as "0.0.1" (I don't think we need to bother with SNAPSHOTs right now) until the first release.